src/components/scripture/dto/scripture-reference.dto.ts

```diff
@@ -12,19 +13,20 @@ export abstract class ScriptureReferenceInput {
   @Field({
     description: 'The code of the Bible book',
   })
-  // TODO validate code against list
   book: string;
 
   @Field(() => Int, {
     description: 'The chapter number',
   })
   @IsPositive()
```

Comment:
Not necessary anymore.
```suggestion
```

Reply:
I will add the logic for a positive number to the validate function. so it can be deleted.